src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -69,11 +70,12 @@ public void configureCoreProperties(
     createShowConfigurationTask();
   }
 
-  private void configureCloudSdkBuilderFactory() {
+  private void configureFactories() {
     project.afterEvaluate(
         project -> {
           // create the sdk builder factory after we know the location of the sdk
           cloudSdkBuilderFactory = new CloudSdkBuilderFactory(toolsExtension.getCloudSdkHome());
+          managedCloudSdkFactory = new ManagedCloudSdkFactory();
```

Comment:
Maybe this could parallel the `CloudSdkBuilderFactory` instantiation by getting `toolsExtension.getCloudSdkVersion()` here.